### PR TITLE
Added the New updated  Programming Courses to keep learners in update.

### PR DESCRIPTION
**Pull request by Cyber-santhosh**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
Added the New updated Course for beginners. 

### Why is this valuable (or not)?
Programming learning updates are valuable as they keep learners abreast of the latest industry trends and techniques. By staying current, learners can adapt to new programming languages and frameworks, enhancing their skill set and employability. 

### How do we know it's really free?
It is available on Youtube

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
We do NOT allow shortened links.

**Comment by Cyber-santhosh** [comment]
> For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes We do NOT allow shortened links.

**Review by sohansai — COMMENTED** [review]
Don't create new titles. Use the same playlist name on YouTube.

**Comment by Cyber-santhosh** [comment]
Changes has been done as per the request the titles has been changed which is same on the Youtube. Pls merge this request using the hacktoberfest label 
Thanks.

File: courses/free-courses-ta.md [hunk 1]
```diff
@@ -109,11 +109,13 @@ CSE Tamila by Eezytutorials
 * [Python DJango in Tamil - Full Course](https://www.youtube.com/playlist?list=PLgWpUXNR_WCch5K1nkemWWsm3rvr-7YmO) - Payilagam
 * [Python in Tamil for Beginners](https://youtube.com/playlist?list=PLA2UBjeRwle3OLO3qmXTbmCvuTlqhHRVb) - GURUKULA
 * [Python Programming in Tami](https://www.youtube.com/playlist?list=PLWbtDrDnmTHBdEnUKuLNdH2-zKSDD8OA4) - CS in Tamil
+* [Python Programming Tutorial Series | In Tamil | You can become a Python Developer.](https://www.youtube.com/playlist?list=PLvepBxfiuao1hO1vPOskQ1X4dbjGXF9bm) - Error Makes Clever
```
Comment (eshellman) [inline courses/free-courses-ta.md:112]:
```suggestion
* [Python Programming Tutorial Series - In Tamil - You can become a Python Developer.](https://www.youtube.com/playlist?list=PLvepBxfiuao1hO1vPOskQ1X4dbjGXF9bm) - Error Makes Clever Academy
```

Comment (Cyber-santhosh) [reply courses/free-courses-ta.md]: Changes has been done as per the suggestion. pls merge my request using hacktoberfest tag.

Thanks.

**Comment by Cyber-santhosh** [comment]
Changes has been done as per the suggestion. Pls merge my request using the hacktoberfest tag.

Thanks.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**